.github/workflows/assign-dependabot.yml

```diff
@@ -0,0 +1,28 @@
+name: Assign Dependabot PRs
+
+on:
+  pull_request:
```

Comment:
Should this specify `types: [created]` to avoid further runs once the initial dispatch is complete?

Reply:
I had doubts about that too but according to the [doc](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#pull_request):
> By default, a workflow only runs when a pull_request event's activity type is opened, synchronize, or reopened

Reply:
Edit: actually `synchronize` is basically a push event sooo yeah fair point 😄

Reply:
hm yes I could probably limit to `opened, reopened` because it seems dependabot can continue pushing commits to an opened branch (even if the label should remain the same so "should" be harmless